src/javascript/EditPanel/EditPanelContent/Preview/Preview.container.jsx

```diff
@@ -26,14 +26,7 @@ export const PreviewContainer = ({isDirty}) => {
     return (
         <Paper className={classes.content}>
             <div className={classes.container}>
-                {isDirty &&
-                    <div>
-                        <Badge className={classes.badge}
-                               badgeContent={t('content-editor:label.contentEditor.preview.updateOnSave')}
-                               variant="normal"
-                               color="info"
-                        />
-                    </div>}
+                <UpdateOnSaveBadge/>
```

Comment:
Moved part dependant on formik in a side component to avoid formik dep and tree rendering